### PR TITLE
Sync with latest layout

### PR DESCRIPTION
**Pull request by richleland**
OK - have a working setup here with app + nginx. Some notes:

* You can clone this repo and run `cookiecutter ~/path/to/cookiecutter-flask-service`
* I added a post-generation hook to automatically copy `.env.template` to `.env`
* You'll need to edit your /etc/hosts to include the new host
* You'll need to edit, at a minimum, the `DOCKER_REGISTRY`, `SSL_KEY_SOURCE_PATH`, and `SSL_CRT_SOURCE_PATH` variables in the `.env` file to point to our Avaaz-specific settings

Task list:

* [x] update folder layout to match latest working services
* [x] set up the .env file automatically
* [x] add docker-compose configs for various environments
* [x] add web server
* [x] put shell scripts in a top-level `scripts` folder
* [x] add mysql database

### CODE REVIEW

File: {{cookiecutter.repo_name}}/app/Pipfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+[packages]
+dataclasses = "*"
+flask = "*"
+flask-cors = "*"
+flask-sqlalchemy = "*"
+geoalchemy2 = "*"
+jinja2 = "*"
+pymysql = "*"
+simplejson = "*"
```
Comment (richleland) [inline {{cookiecutter.repo_name}}/app/Pipfile:14]: Why are we using simplejson and not the built-in json module?

File: {{cookiecutter.repo_name}}/app/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,11 @@
-FROM python:3.6-buster
+ARG AWS_ECR_ENDPOINT
+
+FROM ${AWS_ECR_ENDPOINT}/avaaz/python:latest
```
Comment (richleland) [inline {{cookiecutter.repo_name}}/app/Dockerfile:3]: Note that I removed the default value here - that shouldn't exist in Github

File: {{cookiecutter.repo_name}}/app/Pipfile (hunk 1 above)
Comment (nooravz) [reply {{cookiecutter.repo_name}}/app/Pipfile:14]: simplejson is listed in the flask installation doc here: https://flask.palletsprojects.com/en/1.1.x/installation/ and 
when installed, it is the module preferred/used by flask to:

1. Serialize datetime objects as RFC 822 strings.
2. Serialize objects using their `__html__` method when they have it defined (like Markup).
3. Run the `htmlsafe_dumps` function as a filter called when `|tojson` is used in a jinja template.

simplejson also claims that it is more performant than the built-in module, but I haven't tested that.

File: {{cookiecutter.repo_name}}/app/Dockerfile (hunk 2 above)
Comment (nooravz) [reply {{cookiecutter.repo_name}}/app/Dockerfile:3]: perhaps we should also abstract that further and rename it to something more generic? people might use docker hub as a repo.

Comment (richleland) [reply {{cookiecutter.repo_name}}/app/Dockerfile:3]: yeah good idea

Comment (richleland) [reply {{cookiecutter.repo_name}}/app/Dockerfile]: I made this change and renamed it to `DOCKER_REGISTRY`

**Pull request merged**